### DISCUSSION TITLE
why does Apisix force to configure Keycloak?

### DISCUSSION THREAD

**Discussion by tirelibirefe** [apache/apisix#6773] [Q&A]
Hello,
I'm a newbie for Apisix and trying to adopt environment 
When I attempt to create an apisixroute by using dashboard, apisix forces me configure Keycloak but I don't want to configure Keycloak right now, maybe later. How can I bypass it?

![image](https://user-images.githubusercontent.com/45227339/161163840-e5aea8f1-1428-4e50-b3ee-09de05819612.png)

Thanks & Regards

**Comment by starsz** [comment]
Hi @tirelibirefe .
What's your Apache APISIX-Dashboard version ?

**Reply by tirelibirefe** [reply]
2.13

**Reply by Baoyuantop** [reply]
Hi @tirelibirefe, this warning message comes from the plugin orchestration module of the APISIX Dashboard. Please confirm whether the function is used normally.

**Reply by juzhiyuan** [reply]
I have no idea how to reproduce this step, you could visit the Route List page, Edit that record, then deletes the `scripts` field.

Another thing is, @Baoyuantop I just noticed a typo in the screenshot, see Popup message.

**Reply by Baoyuantop** [reply]
Create a issue for the typo https://github.com/apache/apisix-dashboard/issues/2412

**Reply by tirelibirefe** [reply]
Thanks for all your supports.
I removed Apisix from the system. 
The ticket can be closed (or not).
